test/src/test/java/jenkins/model/JenkinsTest.java

```diff
@@ -472,41 +475,66 @@ public void runScriptOnOfflineComputer() throws Exception {
 
     @Test
     @Issue("JENKINS-38487")
-    public void startupShouldNotFailOnFailingOnlineListener() {
-        // We do nothing, FailingOnOnlineListener & JenkinsRule should cause the 
+    public void startupShouldNotFailOnIOExceptionOnlineListener() {
+        // We do nothing, IOExceptionOnOnlineListener & JenkinsRule should cause the
         // boot failure if the issue is not fixed.
+
+        Assert.assertEquals(1, IOExceptionOnOnlineListener.onOnlineCount);
```

Comment:
```suggestion
        assertEquals(1, IOExceptionOnOnlineListener.onOnlineCount);
```
I suggest following the existing pattern in this file and calling the assert() methods directly. Then the new import also wouldn't be needed.